src/resources/views/store/s3.blade.php

```diff
@@ -14,8 +14,10 @@
         @error('region')
             <p class="help-block">{{$message}}</p>
         @enderror
-        <p class="help-block">Leave empty if your cloud storage service does not support regions.</p>
     </div>
+    @if (!$errors->has('region'))
+        <p class="help-block">Leave empty if your cloud storage service does not support regions.</p>
+    @endif
```

Comment:
Why move this out of the `<div>`? You can also use `@error ... @else ... @enderror`.